aspnet/visual-studio/overview/2013/release-notes.md

```diff
@@ -380,6 +380,9 @@ The following code demonstrates how to enable CORS or JSONP in a SignalR 2.0 pro
 
 ### iOS and Android support via MonoTouch and MonoDroid
 
+> [!IMPORTANT]
```

Comment:
This !Important note was run by the content developer for .NET MAUI first and pre-approved.